snippets/visualbasic/VS_Snippets_Data/XmlWriter.WriteBase64/VB/writebase64.vb

```diff
@@ -43,7 +43,7 @@ Class TestBase64
     ' passed by the user is encoded and included in the document.
     Public Shared Sub EncodeXmlFile(xmlFileName As String, fileOld As FileStream)
 
-        Dim buffer(bufferSize) As Byte
+        Dim buffer(bufferSize - 1) As Byte
```

Comment:
I'll defer to @rpetrusha but I think this should not be changed. (same comment in DecodeOriginalObject.

Reply:
@BillWagner, In Visual Basic, We type the `size - 1` inside the parenthesis, not the `size` as in C#.

For example:

```vb
Sub Main
    Dim buffer(5) As Integer
    Console.WriteLine(buffer.Length) ' Prints: 6
End Sub

```

I actually opened issue #1481 because the array size is wrong in many visual basic code.
Hope I'm not missing something.